examples/mpc-contract/.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "link-arg=-s"]
+
+[build]
```

Comment:
Plz remove the file (for uniformity) It has since been removed for all other examples

Reply:
I removed the build section, but the fil exists in all the examples